app/views/content/train-to-be-a-teacher/if-you-dont-have-a-degree.md

```diff
@@ -32,4 +32,6 @@ Qualifications vary depending on the course. For example, you could get QTS with
 
 [Find out more about the qualifications you need to be a teacher in England](/is-teaching-right-for-me/qualifications-you-need-to-teach).
 
+As part of selecting your teacher training course, you will need to decide if you want to train to teach at a primary or secondary level. [Learn about deciding who to teach](/train-to-be-a-teacher/who-do-you-want-to-teach).
```

Comment:
@philspray @EllieNodder 
This doesn't quite make sense I think - you are choosing a degree rather than a teacher training course?

However, I appreciate we say 'Find your teacher training course' as the CTA below as well. Might be worth checking with policy/UR if this is the right/common terminology for undergraduates?